src/prebid.js

```diff
@@ -171,6 +171,36 @@ function getWinningBidTargeting() {
   return winners;
 }
 
+exports.getWinningBidTargeting = function() {
+  return getWinningBidTargeting(...arguments);
+};
+
+/**
+ * Get custom targeting keys for bids that have `alwaysUseBid=true`.
+ */
+function getAlwaysUseBidTargeting() {
+  return pbjs._bidsReceived.map(bid => {
+    if (bid.alwaysUseBid) {
+      const standardKeys = CONSTANTS.TARGETING_KEYS;
+      return {
+        [bid.adUnitCode]: Object.keys(bid.adserverTargeting, key => key).map(key => {
+          
+          // Get only the non-standard keys of the losing bids, since we
+          // don't want to override the standard keys of the winning bid.
+          if (standardKeys.indexOf(key) > -1) {
```

Comment:
regarding standardKeys.includes - this gave me an error in IE while the previously proposed solution works fine. Therefore, either a polyfill might be needed or indexOf should be left untouched [Stackoverflow covers this as well](http://stackoverflow.com/questions/31119300/ie11-object-doesnt-support-property-or-method-includes-javascript-window)